packages/account/src/Sections/Security/AccountClosed/account-closed.tsx

```diff
@@ -20,7 +23,7 @@ const AccountClosed = ({ logout }) => {
     }, [is_appstore, timer]);
 
     React.useEffect(() => {
-        history.push(routes.root);
+        window.history.pushState(null, '', '/');
```

Comment:
```suggestion
        window.history.pushState(null, '', '/');
```
Why not use useHistory from react-router?

Reply:
As discussed with @yashim-deriv, this will be handled later during routes refactoring.